create.html

```diff
@@ -14,59 +14,59 @@
     <title>Quizzzle</title>
   </head>
   <body>
-    <header class="zz-top">
+    <header class="zz-top title">
       <h1>Create</h1>
     </header>
 
-    <main>
+    <main class="centered">
       <span class="sr-only" id = 'form-label'>Create a new Question</span>
       <form aria-labelledby="form-label" action="" class="submit-form">
```

Comment:
I'd go with the classname `form` (because every form is a "submit-form"). It's a bit like calling a `button` a `click-button`.